snippets/promisify.md

```diff
@@ -8,9 +8,7 @@ const promisify = func =>
   (...args) =>
     new Promise((resolve, reject) =>
       func(...args, (err, result) =>
-        err
-        ? reject(err)
-        : resolve(result))
+        err ? reject(err) : resolve(result));
```

Comment:
I had to remove this semicolon and add it at [L12](https://github.com/Chalarangelo/30-seconds-of-code/pull/64/commits/f6c181e0f4e03d523070448dae937455f0a82134#diff-a86c09b8ca7cbea118d447f7e3cf705dR12).